src/big/big_rational.cr

```diff
@@ -367,15 +367,15 @@ struct BigRational < Number
     denominator.format(io, separator, delimiter, decimal_places, group: group, only_significant: only_significant)
   end
 
-  def clone
+  def clone : BigRational
     self
   end
 
   private def mpq
```

Comment:
Why are some of these functions not typed at all?

Should be `: LibGMP::MPQ*`

Reply:
The `cr-source-typer` has a new flag to ignore private defs, as well as type restrictions that are resolved to 3+ union types.

I realize my PR description doesn't show that with the single line command :grimacing: